<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceDialog</class>
 <widget class="QDialog" name="ResourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QComboBox" name="resourceTypeComboBox"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="resourceIdLineEdit">
      <property name="styleSheet">
       <string notr="true">background: rgb(220, 220, 220);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="userPasswordLabel">
      <property name="text">
       <string>Mot de passe (*)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="userLoginLabel">
      <property name="text">
       <string>Identifiant (*)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="resourceNameLineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="resourceFirstnameLabel">
      <property name="text">
       <string>Prénom (*)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="resourceTypeLabel">
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="resourceFirstnameLineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="resourceNameLabel">
      <property name="text">
       <string>Nom (*)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="resourceIdLabel">
      <property name="text">
       <string>Numéro</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="userLoginLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="userPasswordLineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="resourceDialogHeader">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fiche ressource</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>164</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelResourceEditionButton">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="validateResourceEditionButton">
      <property name="text">
       <string>Valider</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>resourceFirstnameLineEdit</tabstop>
  <tabstop>resourceNameLineEdit</tabstop>
  <tabstop>resourceTypeComboBox</tabstop>
  <tabstop>resourceIdLineEdit</tabstop>
  <tabstop>userLoginLineEdit</tabstop>
  <tabstop>userPasswordLineEdit</tabstop>
  <tabstop>validateResourceEditionButton</tabstop>
  <tabstop>cancelResourceEditionButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
